src/app/auth/login/page.jsx

```
@@ -134,180 +137,364 @@
+                            Enter your email to receive reset instructions
+                          </p>
+                        </div>
+
+                        <form
+                          onSubmit={handleForgotPassword}
+                          className="space-y-4"
+                        >
+                          <div>
+                            <label className="block text-sm font-medium text-gray-300 mb-2">
+                              Email Address
+                            </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)